index.bs

```diff
@@ -12442,6 +12441,12 @@ object is associated with.
                 1.  Let |interfaceObject| be the result of [=create a named constructor|creating
                     a named constructor=] with |id| for |interface| in |realm|.
                 1.  Perform [=!=] [$CreateMethodProperty$](|target|, |id|, |interfaceObject|).
+    1.  [=list/iterate|For every=] [=callback interface=] |interface| that is [=exposed=] in
```

Comment:
"Define the interface objects" is no longer a good name for this algorithm, as it creates interface objects, legacy callback interface objects, and (in the next commit) namespace objects. Maybe "Define the top-level constructs" or "Define the global property references" or something.